spec/services/bulk_upload/lettings/year2023/row_parser_spec.rb

```diff
@@ -86,8 +86,6 @@
     before do
       stub_request(:get, /api\.postcodes\.io/)
       .to_return(status: 200, body: "{\"status\":200,\"result\":{\"admin_district\":\"Manchester\", \"codes\":{\"admin_district\": \"E08000003\"}}}", headers: {})
-
-      parser.valid?
```

Comment:
This was causing ordering issues, since it gets called before some setup done in nested contexts. I've moved to calling it in the tests themselves so that all setup has happened first.